### PR TITLE
[Nokia]: Enable Telemetry for armhf and provide required qos files

### PR DESCRIPTION
**Pull request by carl-nokia**
**- Why I did it**
To enable Telemetry on armhf architecture and provide required qos/buffers files that 
config qos reload --no-dynamic-buffer now requires for the Nokia ixs7215 

**- How I did it**
Removed the hardcoded disable of Telemetry in slave.mk for armhf architecture and provided 
required qos/buffers files for the config qos reload code to succeed as stubs since they do not 
generate buffer settings to Marvell SAI for this MO based role

**- How to verify it**
config load_minigraph -y succeeds for OC testing 

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by qiluo-msft** [comment]
The PR title is not descriptive.

**Comment by carl-nokia** [comment]
Fixed the title - was an oversight. Thanks

File: slave.mk [hunk 1]
```diff
@@ -369,7 +367,6 @@ SONIC_TARGET_LIST += $(addprefix $(FILES_PATH)/, $(SONIC_ONLINE_FILES))
 #     $(SOME_NEW_FILE)_DEPENDS = $(SOME_OTHER_DEB1) $(SOME_OTHER_DEB2) ...
 #     SONIC_MAKE_FILES += $(SOME_NEW_FILE)
 $(addprefix $(FILES_PATH)/, $(SONIC_MAKE_FILES)) : $(FILES_PATH)/% : .platform $$(addsuffix -install,$$(addprefix $(DEBS_PATH)/,$$($$*_DEPENDS))) \
-			$$(addprefix $(DEBS_PATH)/,$$($$*_AFTER)) \
```
Comment (lguohan) [inline slave.mk:372]: i think this is unintentional, can you revert this change?

**Review by lguohan — CHANGES_REQUESTED** [review]
can you revert the change?

**Comment by carl-nokia** [comment]
Reverted the commit with mistake and pushed the one line change to slave.mk
Please take a look - Thank You

**Review by lguohan — APPROVED** [review]

**Comment by carl-nokia** [comment]
Retest mellanox please

**Comment by lguohan** [comment]
build test failure irrelavant
